### ISSUE TITLE
Provide an updated release

### ISSUE THREAD

**Issue by TheFrenchGhosty** [snwh/paper-icon-theme#750]
It would be nice if you could make a new release with the latest change.

Right now release have some broken icons, for example XFCE / Thunar:

![screenshot](https://user-images.githubusercontent.com/47571719/84306669-4fec3b00-ab4b-11ea-9a8a-123b6f472182.png)

Those broken icons are not in the latest git version.

**Comment by robojerk** [comment]
I'm running Fedora 32 Cinnamon
Same issue in Nemo and Thunarr.

**Comment by Starcr0ss** [comment]
I run Pop!_OS 20.04 LTS, and suffer with these problems in all GTK file managers and other GTK apps that use the forward and back arrows.

**Comment by robojerk** [comment]
If you build the theme, it works....
I think the package maintainers for the various distros need to update the buid..

OR you could use [Papirus](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme).  It's based off of Paper and gets regular updates.

**Comment by LinuxOnTheDesktop** [comment]
Relatedly: the Paper PPA seems not to support Ubuntu Focal - and, thus, does not support any of the distros (e.g. Mint 20) based on Ubuntu Focal.

**Comment by TheFrenchGhosty** [comment]
> If you build the theme, it works....
> I think the package maintainers for the various distros need to update the buid..
> 
> OR you could use [Papirus](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme). It's based off of Paper and gets regular updates.

Papirus is far from Paper even if it's based on it.

**Comment by LinuxOnTheDesktop** [comment]
Yes: Papirus is more cartoon-ish than Paper and also . . it's _blue_.

If distro package maintainers for the various distros need to update the build, then I imagine it is up to Sam Hewitt to notify those maintainers. Perhaps he did not do so. I remain grateful for his work.

**Comment by Pablorg99** [comment]
> Relatedly: the Paper PPA seems not to support Ubuntu Focal - and, thus, does not support any of the distros (e.g. Mint 20) based on Ubuntu Focal.

Exactly. Maybe we should open another issue for this? @snwh

I want these amazing icons with updates on my recently updated Ubuntu 20.04  

EDIT: There is already an issue for this #744

**Comment by StephenBeirlaen** [comment]
The packages are now available for ubuntu20.10.1 and ubuntu20.04.1 since 2020-07-19
https://launchpad.net/~snwh/+archive/ubuntu/ppa

**Comment by LinuxOnTheDesktop** [comment]
Marvellous. (I had to re-add the PPA to my system.) So, this issue can be closed.

**Comment by tadfisher** [comment]
A release should still be tagged, as many distros use Git tags to notify for updates, and as a signal that the release is stable. NixOS, for example, will automatically build and check its package when a tag is pushed.
